compiler/rustc_resolve/src/lib.rs

```diff
@@ -1105,7 +1105,8 @@ impl<'a> AsMut<Resolver<'a>> for Resolver<'a> {
 }
 
 impl<'a, 'b> DefIdTree for &'a Resolver<'b> {
-    fn parent(self, id: DefId) -> Option<DefId> {
+    #[inline]
```

Comment:
Does the use of `#[inline]` here actually improve perf?

Reply:
I guess we could start with a perf run on this PR first. I don't know if I would be available when CI passes. Could you initiate a perf run? Thanks.